cxbox-core/src/main/java/org/cxbox/core/dao/impl/MetadataUtils.java

```diff
@@ -229,15 +241,7 @@ public static Predicate createPredicate(Root<?> root, ClassifyDataParameter crit
 				case GREATER_OR_EQUAL_THAN:
 					return cb.greaterThanOrEqualTo(field, requireComparable(value));
 				case LESS_OR_EQUAL_THAN:
-					return cb.lessThanOrEqualTo(field, requireComparable(value));
-				case INTERVALS:
-					return cb.or(((List<Period>) value).stream()
-							.map(object ->
-									cb.and(
-											cb.greaterThanOrEqualTo(field, requireComparable(object.getStart())),
-											cb.lessThanOrEqualTo(field, requireComparable(object.getEnd()))
-									))
-							.toArray(Predicate[]::new));
+					return cb.greaterThanOrEqualTo(field, requireComparable(value));
```

Comment:
why INTERVALS deleted and why LESS_OR_EQUAL_THANproduces cb.greaterThanOrEqualTo